### ISSUE TITLE
RRM 508 Compliance Project

### ISSUE THREAD

**Issue by merrellc** [ONRR/onrr.gov-site#520]
eCommerce Online Rental Payments - Document has been updated.

Location of document - 

https://www.onrr.gov/ReportPay/PDFDocs/eCommerce -Online-Rental-Payments-FAQ.pdf
--

Thank you

[eCommerce -Online-Rental-Payments-FAQ (9).pdf](https://github.com/ONRR/onrr.gov-site/files/5998287/eCommerce.-Online-Rental-Payments-FAQ.9.pdf)

**Comment by merrellc** [comment]
Attached is the updated Transaction Code document
Location for document- 
[TranCodes.pdf](https://github.com/ONRR/onrr.gov-site/files/5998620/TranCodes.pdf)

https://www.onrr.gov/ReportPay/PDFDocs/TranCodes.pdf
--

**Comment by jennmalcolm** [comment]
Thanks @merrellc I have posted these two corrected documents to dev and the live site. Do you want to leave this issue open so you can add more documents?

**Comment by merrellc** [comment]
Yes, please leave open.


Thank you
​Candace Merrell
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resources Revenue
303-231-3517
Candace.Merrell@onrr.gov

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.

________________________________

**Comment by bowlenj** [comment]
Please delete this 2019 outdated pdf contact list - https://www.onrr.gov/ReportPay/PDFDocs/operasgn.pdf

Replace Onshore Production Reporting with attached updated pdf
[RRM Onshore Production Contacts.pdf](https://github.com/ONRR/onrr.gov-site/files/6045594/RRM.Onshore.Production.Contacts.pdf)
 - https://www.onrr.gov/about/contact.htm

**Comment by bowlenj** [comment]
Replace Form ONRR-4444 Instructions located at - https://www.onrr.gov/reportpay/PDFDocs/ONRR4444_Instructions.pdf

Replace with attached updated ONRR-4444 Instructions
[ONRR4444_Instructions.pdf](https://github.com/ONRR/onrr.gov-site/files/6045610/ONRR4444_Instructions.pdf)

**Comment by bowlenj** [comment]
Please delete the below content located at - https://www.onrr.gov/ReportPay/CrossRef.htm.  This information is outdated and is already being uploaded under the Leases and Agreements above it, which is already compliant:

Agreements Cross-Referenced to former Accounting Identification (AID) Numbers

These cross-referenced files show all former Accounting Identification (AID) Numbers that identified production allocated from an agreement. The AID numbers were used prior to October 1, 2001 and consisted of the 10-digit ONRR lease number and the 3-digit revenue source1 number. AID Numbers that identified lease-basis production are not included in these files. 
Please Note: Some lines on these reports are duplicates and some lines contain a Revenue Source of 000. Please ignore these lines.
•Cross-Referenced AID and Agreement Numbers Sorted by Agreement Name 
•Cross-Referenced AID and Agreement Numbers Sorted by Agreement Number 
•Cross-Referenced AID and Agreement Numbers Sorted by Lease 
•Downloadable Excel Version 

**Comment by bowlenj** [comment]
Replace ACH Instructions located at - https://www.onrr.gov/ReportPay/PDFDocs/ACHInst.pdf

Replace with attached updated ACH Instructions
[ACH Instructions.pdf](https://github.com/ONRR/onrr.gov-site/files/6045638/ACH.Instructions.pdf)

**Comment by mcharg** [comment]
@bowlenj The 4444 instructions should be updated.  I will update the others tomorrow.

**Comment by mcharg** [comment]
@bowlenj The change to the cross-refence page is on dev.  Can you please review before I push to the live site?
http://2017onrr/reportpay/CrossRef.htm

**Comment by bowlenj** [comment]
Yes, this looks great!  Thank you 😊

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resource Revenue
303-231-3202
Jessica.Bowlen@onrr.gov

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.

**Comment by mcharg** [comment]
@bowlenj The cross-reference and ACH changes should be updated on the live site.  Can I close this issue now or do you want to keep it open to add additional docs?

**Comment by bowlenj** [comment]
You can close it, thank you.  I think I'm done with those, just waiting for corrections to the Onshore Production Contact list.

Thank You,
Jessica Bowlen
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resource Revenue
303-231-3202
Jessica.Bowlen@onrr.gov

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.
